### PR TITLE
Restructure the code and the changed API document

### PR DESCRIPTION
**Pull request by yashodgayashan**

### CODE REVIEW

**Comment by yashodgayashan** [comment]
@deshankoswatte @sanjulamadurapperuma @vikumkbv  I have regenerate the code and change the API document names as requested. Please go through the method names and suggest if there should be any changes.

File: core/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target
```
Comment (deshankoswatte) [inline core/.gitignore:1]: Add new line at the end of file.

File: core/Ballerina.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [project]
-org-name= "environmentplatform"
+org-name= "yashodperera"
```
Comment (deshankoswatte) [inline core/Ballerina.toml:2]: Revert to the previous name.
```suggestion
org-name= "environmentplatform"
```

File: core/src/env_platform/schema.bal [hunk 3]
```diff
@@ -91,4 +91,4 @@ type Field record {
      Ministry addedBy;
      Date addedOn;
      boolean edited;
-};
+};
```
Comment (deshankoswatte) [inline core/src/env_platform/schema.bal:94]: Add new line at the end of file.

File: core/Ballerina.toml (hunk 2 above)
Comment (sanjulamadurapperuma) [reply core/Ballerina.toml:2]: @deshankoswatte @yashodgayashan what do you think, shall we go ahead with just environmentplatform or any ideas for any other names like environmentplatformsl or something similar?

Comment (deshankoswatte) [reply core/Ballerina.toml:2]: It's a +1 from me fr anything which is suitable including what @sanjulamadurapperuma mentioned. @yashodgayashan thoughts?

**Comment by sanjulamadurapperuma** [comment]
> Requested some changes. @yashodgayashan where is the `Location` object/entity/schema?

Location is there

**Comment by yashodgayashan** [comment]
> Requested some changes. @yashodgayashan where is the `Location` object/entity/schema?

Since it is not a object it is not auto generated I will add it.

File: core/Ballerina.toml (hunk 2 above)
Comment (sanjulamadurapperuma) [reply core/Ballerina.toml]: @yashodgayashan so shall we keep this as environmentplatform?

**Comment by deshankoswatte** [comment]
@yashodgayashan few more suggestions. Try to protect the master branch to required two approvals for a PR and also remove the .DS_Store file from the master branch since it is generated from macOS.

**Comment by yashodgayashan** [comment]
@vikumkbv Waiting for your apporval.

**Pull request merged**